### ISSUE TITLE
Add quick way to fully delete a member

### ISSUE THREAD

**Issue by nosegraze** [restrictcontentpro/restrict-content-pro#1177]
I get this support question a lot. People have to delete the user account in the "Users" menu, which isn't really very intuitive. It would be nice to have an option for this in the RCP Members table (maybe a bulk action and a row action).

**Comment by mindctrl** [comment]
We've had several people want to delete the membership from a user without deleting the user account.

**Comment by braddalton** [comment]
Agree. Add a bulk action to Mark as Expired and Mark as Cancelled in the drop down for deleting pending memberships.

**Comment by nosegraze** [comment]
WIP PR: #1712

For now just added the functionality for deleting the account. Will add in the option to remove membership after we get in the new subscriptions table.
